tests/unit/lib/charms/oai_ran_cu_k8s/v0/test_fiveg_f1_provider.py

```diff
@@ -4,8 +4,8 @@
 from unittest.mock import patch
 
 import pytest
-import scenario
```

Comment:
Should we also remove the dependency on scenario in the requirements file?

Reply:
No, `ops-scenario` is an optional extra that needs to be installed on its own.

Reference:
- https://github.com/canonical/operator/blob/main/ops/testing.py#L81